src/components/policies/policies.tsx

```diff
@@ -0,0 +1,98 @@
+import { useMemo } from 'react';
```

Comment:
This is the previous `src/pages/policies/policies.tsx` file that GH has decided to pick up as a new file. The only difference is that it receives as props:
```ts
  query: string | null;
  filter: Filter;
  currentPage: number;
```